src/cli.coffee

```diff
@@ -35,6 +35,7 @@ watchCmd = program
   .option('-p, --port [port]', 'if a `server` option was specified, define on which port
  the server would run')
   .option('-d, --debug', 'print verbose debug output to stdout')
+  .option('-l, --stdin', 'listen to stdin and shut down when stdin closes')
```

Comment:
oh ok, I had taken your word for it before that it would work. So I suppose we should just leave it like this, or just remove the shorthand flag.

@josevalim why did you choose `-l`?